www/src/views/shared/header.ejs

```diff
@@ -35,6 +35,9 @@
                    <a ng-repeat="theme in themes" class="dropdown-item" href="" ng-click="setTheme(theme)">{{::theme}}</a>
                 </div>
 			</li>
+			<li class="nav-item">
+				<a class="nav-link" href="http://legendmud.org:19999/" target="_blank">Play Now</a>
```

Comment:
When linking to external sites, in addition to `target="_blank"`, I also like to add `rel="noopener noreferrer"`. This is for SEO purposes.